File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/non-plagiarized/14/Soal5.ts
import { Scanner } from 'readline-sync';

function reverse(number: number): void {
  const a: string = number.toString();
  
  for (let i: number = a.length - 1; i > -1; i--) {
    process.stdout.write(a.charAt(i));
  }
  console.log('');
}

const sc: Scanner = new Scanner();
console.log('Enter an integer: ');
const number: number = sc.nextInt();
reverse(number);

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/non-plagiarized/14/Soal5.kt
package IR

import java.util.Scanner

/**
 *
 * @author 4900D0A19B6894A4A514E9FF3AFCC2C0
 */
fun reverse(number: Int) {
    val a = number.toString()
    for (i in a.length - 1 downTo 0) {
        print(a[i])
    }
    println()
}

fun main() {
    val sc = Scanner(System.`in`)
    print("Enter an integer: ")
    val number = sc.nextInt()
    reverse(number)
}

